pnet_transport/Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["networking", "transport", "raw", "socket"]
 categories = ["network-programming"]
 
 [dependencies]
-libc = "0.2"
+libc = ">=0.2.34"
```

Comment:
No real need to bump this version. We only need `0.2.34` for the changes in `pnet_sys`. Nothing in `pnet_transport` explicitly requires the newer `libc`.

If it is left bumped, then please see my other comment on caret requirements.